tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/AspNetCoreTestFixture.cs

```diff
@@ -39,6 +39,8 @@ public AspNetCoreTestFixture()
 
         public int HttpPort { get; private set; }
 
+        public string ShutdownPath { get; set; }
```

Comment:
Given that with removal of this path from the constructor means it's now _optional_ I wonder if we should set a default here.

My thinking is that it should be easier to spot an _incorrect_ shutdown path (if that path doesn't exist, for example) than the app being left often (which will likely cause seemingly unrelated issues?)

```suggestion
        public string ShutdownPath { get; set; } = "/home/shutdown";
```